atomist/configure/advisories.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Advisory sources
+description:
+keywords:
+---
+
+{% include atomist/disclaimer.md %}
+
+Vulnerabilities and advisories in the Atomist vulnerability database are updated
+continuously. Data from advisories is ingested, processed, enriched, and
+transformed from the following sources, by default:
```

Comment:
I didn't think to question whether we do, since the same content is currently published for the old documentation site, too. Just not included in the toc: <https://docs.atomist.com/getting_started/sources/>

Should I remove it from here?